Documentation/git-config.txt

```diff
@@ -235,8 +235,12 @@ FILES
 If not set explicitly with `--file`, there are four files where
 'git config' will search for configuration options:
 
+$PROGRAMDATA/Git/config::
+	System-wide configuration file for *all* Git-related software running
```

Comment:
@nalla The "reading" part mentions that files are read in top-down order but for writing to --global (where already two files exist) the priority is bottom-up. I just assumed the same logic for two --system files.